openedx/core/djangoapps/notifications/views.py

```diff
@@ -479,3 +482,13 @@ def patch(self, request, *args, **kwargs):
             return Response({'message': _('Notifications marked read.')}, status=status.HTTP_200_OK)
 
         return Response({'error': _('Invalid app_name or notification_id.')}, status=status.HTTP_400_BAD_REQUEST)
+
+
+@api_view(['GET', 'POST'])
+def preference_update_from_encrypted_username_view(request, username, patch):
```

Comment:
Add docs about what params are , i.e what are required fields in patch .  and also check if it is rendering properly in API docs

Reply:
Added docs in the function which is encrypting data. Docs are not needed for API as it is using encrypted username and patch.